osidb/mixins.py

```diff
@@ -148,68 +149,79 @@ class Meta:
         abstract = True
 
 
+class Alert(models.Model):
```

Comment:
@dmonzonis I was thinking about using the alerts in the collector framework to store some data collection issues so we can query them easily. While thinking about it I realized one issue here.

The alerts do not have any ACL controls while they can still be freely queried on API by anyone authenticated. That means that even though a user would have only a public access-level, (s)he could eg. see alerts of the embargoed flaws and OSIDB would be leaking an information about their existence and potentially much more. The embargo is always taken quite seriously in ProdSec so I would see it as quite an issue.

I see two way how to overcome it. Probably easier way would be to restrict the API endpoint to those with the embargo access only. Anyone could see instance-specific alerts among its data so this general endpoint would have more narrowed usage anyway. A cleaner solution would be to provide the `Alert` model with ACLs which would be inherited from the parent instance.

I know this is quite late in game so feel free to create a separate Jira for this improvement and plan it for the next sprint so it does not interfere with your already planned work.

Reply:
I changed it so that alerts now properly use the parent's ACLs.